src/dodal/devices/eiger.py

```diff
@@ -39,6 +39,7 @@ def set(self, value, *, timeout=None, settle_time=None, **kwargs):
 
     STALE_PARAMS_TIMEOUT = 60
     GENERAL_STATUS_TIMEOUT = 10
+    META_FILE_READY_TIMEOUT = 30
```

Comment:
Nit: Might be nice to have a comment here about file system issues requiring this